webconsole/gogo/src/main/java/org/apache/karaf/webconsole/gogo/Terminal.java

```diff
@@ -1710,14 +1710,14 @@ public synchronized String pipe(String d) {
                 vt100_keyfilter_escape = true;
             } else if (c == 127) {
                 if (vt100_mode_backspace) {
-                    o += (char) 8;
+                    o += Integer.toString(8);
```

Comment:
It's a `String` whereas we cast to `char`.

Reply:
yes but the char is then autocasted to String. I think that we can get rid of this step, isn't it?

Reply:
Agree, it's just a personal note for the merge ;)